evals/calibrate_judge.py

```
@@ -0,0 +1,333 @@
+
+
+def _pdf_size_mb(url: str) -> Optional[float]:
+    """HEAD a URL and return its size in MB, or None if unknown.
+
+    Uses a curl User-Agent: the default ``Python-urllib`` UA is blocked by the Cloudflare
+    WAF in front of the asset host, which would otherwise make every probe silently fail.
+    """
+    try:
+        req = urllib.request.Request(
+            url, method="HEAD", headers={"User-Agent": "curl/8.4.0"}
+        )
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)